pages/documentation/guidance/accessibility.md

```diff
@@ -6,83 +6,167 @@ redirect_from:
 layout: styleguide
 title: Accessibility
 category: How to use USWDS
-lead: Accessibility tips to help you deliver delightful experiences to all users.
+lead: Accessibility is usability for every ability
```

Comment:
Missing an ending period :)

Reply:
I wouldn't add a period here - I would just change this deck to read "[DELETE Accessibility] Usability for every ability"

Reply:
Some have periods like [How to Use](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.app.cloud.gov/preview/uswds/uswds-site/jm-update-a11y-page/how-to-use-uswds/) and others like [Website Standards](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.app.cloud.gov/preview/uswds/uswds-site/jm-update-a11y-page/website-standards/) and [Quick start](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.app.cloud.gov/preview/uswds/uswds-site/jm-update-a11y-page/documentation/fundamentals/) don't.